ping.py

```diff
@@ -63,19 +65,121 @@ def get_ident(): return 0
 MAX_SLEEP = 1000
 
 
-class MStats:
-    thisIP = "0.0.0.0"
-    pktsSent = 0
-    pktsRcvd = 0
-    minTime = 999999999
-    maxTime = 0
-    totTime = 0
-    avrgTime = 0
-    fracLoss = 1.0
+# noinspection PyAttributeOutsideInit
```

Comment:
It would be...prefered?

I know the rest of the contributing community doesn't much care, but it just seems obtuse to me.

I've been trying to clean up this code here and there when I can and trying to get the rest of the community both active and submitting clean code.